modules/oc-mirror-dry-run-v2.adoc

```diff
@@ -15,19 +15,22 @@ Verify your image set configuration by performing a dry run without mirroring an
 +
 [source,terminal]
 ----
-$ oc mirror -c <image_set_config_yaml> --from file://<oc_mirror_workspace_path> docker://<mirror_registry_url> --dry-run --v2
+$ oc mirror -c <image_set_config_yaml> file://<oc_mirror_workspace_path> --dry-run --v2
 ----
++
 Where:
+
 - `<image_set_config_yaml>`: Use the image set configuration file that you just created.
+
 - `<oc_mirror_workspace_path>`: Insert the address of the workspace path.
-- `<mirror_registry_url>`: Insert the URL or address of the remote container registry from which images will be deleted.
+
+- `<mirror_registry_url>`: Insert the URL or address of the remote container registry from which images will be mirrored or deleted.
 +
 .Example output
 [source,terminal]
 ----
 $ oc mirror --config /tmp/isc_dryrun.yaml file://<oc_mirror_workspace_path> --dry-run --v2
 
```

Comment:
Didn't realize this before but if this is example output, the actually command needs to be removed from the source block. Example output blocks should only have the actual output:
```suggestion
```